### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Stylix58**

### CODE REVIEW

**Comment by Tronic247** [comment]
@Stylix58 No need to add a search :D Algolia docsearch is there.

File: mkdocs.yml [hunk 1]
```diff
@@ -9,5 +9,6 @@ copyright: Tronic247
 theme:
     name: null
     custom_dir: 'theme'
+    include_search_page: true
```
Comment (Tronic247) [inline mkdocs.yml:12]: No need to add a search :D

**Review by Tronic247 — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (Stylix58) [reply mkdocs.yml:12]: its the search included by mkdocs, see https://www.mkdocs.org/dev-guide/themes/#search-and-themes

Comment (Stylix58) [reply mkdocs.yml:12]: so, i will remove algolia docsearch

**Comment by Stylix58** [comment]
You can join me on [gitter](https://gitter.im/Tronic247-Material/community)?

**Pull request closed without merging**